tests/baselines/reference/privateNameJsBadAssignment.errors.txt

```diff
@@ -0,0 +1,30 @@
+tests/cases/conformance/classes/members/privateNames/privateNameJsBadAssignment.js(1,9): error TS2339: Property '#nope' does not exist on type 'typeof import("tests/cases/conformance/classes/members/privateNames/privateNameJsBadAssignment")'.
+tests/cases/conformance/classes/members/privateNames/privateNameJsBadAssignment.js(3,13): error TS18016: Private identifiers are not allowed outside class bodies.
+tests/cases/conformance/classes/members/privateNames/privateNameJsBadAssignment.js(6,3): error TS2339: Property '#foo' does not exist on type 'typeof B'.
+tests/cases/conformance/classes/members/privateNames/privateNameJsBadAssignment.js(12,14): error TS2339: Property '#foo' does not exist on type 'C'.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNameJsBadAssignment.js (4 errors) ====
+    exports.#nope = 1;           // Error (outside class body)
+            ~~~~~
+!!! error TS2339: Property '#nope' does not exist on type 'typeof import("tests/cases/conformance/classes/members/privateNames/privateNameJsBadAssignment")'.
+    function A() { }
+    A.prototype.#no = 2;         // Error (outside class body)
+                ~~~
+!!! error TS18016: Private identifiers are not allowed outside class bodies.
+    
+    class B {}
+    B.#foo = 3;                  // Error (outside class body)
+      ~~~~
+!!! error TS2339: Property '#foo' does not exist on type 'typeof B'.
+    
+    class C {
+        #bar = 6;
+        constructor () {
+            exports.#bar = 6;    // Error

```

Comment:
@sandersn  there should be an error on this line.

There **is** an error on this line when I:
- run the test individually
- try in the language service
- run the code directly by dropping it in a JS file and doing `node ./built/local/tsc.js <file>.js--allowJs --checkJs --noEmit`

The error not showing up here seems to be a weird artifact of the bulk test driver.

What would you recommend? I could delete the line from the test, but don't want to pass over this weird thing in silence in case someone else sees the (potential) issue with the test driver.

cc @rbuckton for thoughts

Reply:
problem went away when I made other changes. I could have been doing something wrong.